.copilot-collections.yaml

```diff
@@ -0,0 +1,8 @@
+copilot:
+  # The version of the toolkit to use (matches a Release Tag in this repo)
+  version: "v0.3.0"
+
+  # The collections you want to install
+  collections:
+    - charm-python
+    - pfe-charms
```

Comment:
It could be a risk in the future to use those profiles, as this repo hosts charm + application level code, and guidelines applicable to charm code might not be useful for application code. We will need to review the instructions in the future, whether they still make sense.  For now, I don't see an issue but rather a benefit of using those profiles.